recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.16.imx.bb

```diff
@@ -36,12 +36,15 @@ DEPENDS += "gstreamer1.0-plugins-base"
 
 inherit gobject-introspection
 
+PACKAGECONFIG_GL ?= "${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'gl', '', d)}"
+PACKAGECONFIG_GL_imxpxp = "gles2"
```

Comment:
The commit log suggests this is a change only for 6UL and 6ULL, but this override `_imxpxp` will change **all** of the 6s and 7s except for 6Q. Perhaps you want `_mx6ul` and `_mx6ull`?

Reply:
I have not tested this on the other platforms, I think you are right to limit this change only on these two platforms, while waiting to test for the others.

Reply:
@CarolZhuYuqing, can you have a look at this as well?

Reply:
@thochstein Do you want me to update my commit with your proposal or you will do that later?

Reply:
Hi @CarolZhuYuqing, what do you think of that?

Reply:
hi, is there any feedback ? I can't see any comment from @CarolZhuYuqing